zebedee-cms/src/main/java/com/github/onsdigital/zebedee/util/slack/NopSlackClientImpl.java

```diff
@@ -0,0 +1,31 @@
+package com.github.onsdigital.zebedee.util.slack;
+
+import com.github.onsdigital.slack.Profile;
+import com.github.onsdigital.slack.client.PostMessageResponse;
+import com.github.onsdigital.slack.client.SlackClient;
+import com.github.onsdigital.slack.messages.PostMessage;
+
+import static com.github.onsdigital.zebedee.logging.CMSLogEvent.info;
+
+/**
+ * No op implementation of {@link SlackClient}
+ */
+public class NopSlackClientImpl implements SlackClient {
+
+    @Override
+    public PostMessageResponse sendMessage(PostMessage postMessage) {
+        info().data("message", postMessage.getText()).log("NopSlackClientImpl.sendMessage");
+        return null;
```

Comment:
I think returning null in these methods could cause a null pointer exception, as the return value is used in some cases. Consider if this will cause any problems.